packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -182,7 +183,10 @@ class CallContext {
         page: newPage,
         endedCall: latestEndedCall,
         devices: clientState.deviceManager,
-        latestErrors: clientState.latestErrors
+        latestErrors: clientState.latestErrors,
+        isCameraOn:
+          clientState.deviceManager.unparentedViews.find((s) => s.mediaStreamType === 'Video') !== undefined ||
+          this.getState().call?.localVideoStreams.find((s) => s.mediaStreamType === 'Video') !== undefined
```

Comment:
dont use `getState()` here, just use `call` that we grab above to be consistent
```suggestion
          call?.localVideoStreams.find((s) => s.mediaStreamType === 'Video') !== undefined
```